earthaccess/auth.py

```diff
@@ -317,60 +271,27 @@ def _get_credentials(
             self.username = username
             self.password = password
 
-            self.tokens = token_resp.json()
+            token = token_resp.json()
+            logger.debug(
+                f"Using token with expiration date: {token['expiration_date']}"
+            )
+            self.token = token
+            self.authenticated = True
```

Comment:
Oops! This line is duplicated. Probably my fault due to how I constructed the code suggestion.

Reply:
Oops, didn't notice